nestor_api/utils/logger.py

```diff
@@ -1,21 +1,66 @@
 """A simple logger implementation awaiting for proper implementation"""
 
+import json
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(levelname)s-%(asctime)s: %(message)s",
+    datefmt="%d-%b-%y %H:%M:%S",
+)
+
 
 class Logger:
-    """A fake logger module to be able to collect logs later on"""
+    """A logger module"""
 
     @staticmethod
-    def debug(context: dict, message: str):
-        """Produce a log with debug level"""
+    def debug(context=None, message=""):
```

Comment:
Do you think it would be better to have the message first ? (and non-optional) so that we could write `Logger.info("message without context")`.
Or disallow the default values and always provide both when calling the logger ?

> I realize switching the arguments might not be compatible with the already written messages that use the current order (context, message)

Reply:
Also --> don't forget to add the typings 😉

Reply:
hihi, added the typings. I always forget about it. We agreed only for the prototype of the function right?